### ISSUE TITLE
Set specific icon for node

### ISSUE THREAD

**Issue by noorbakerally** [wix/angular-tree-control#277]
Is it possible to set a specific icon for a node ?

**Comment by shenghou** [comment]
Yes, you can add specific icon or other what you want , like this.
`
 <treecontrol class="tree-boot"
              tree-model="treeData"
              options="treeOptions"
              expanded-nodes="expandedNodes"
              on-selection="seleNode(node,$index,$first,$last,$path,selected,expanded,$result)"
              selected-node="groupActive">
              <span title="{{node.description}}">
                <span ng-switch="" on="node.taxonomy">
                   <span ng-switch-when="terminalgroup" class="fa fa-folder-open" aria-hidden="true"></span>
                </span>
                <span>{{node.name}}</span>
                 // badge
                <span class="badge clt-bg-success" ng-if="node.onlineNum>0">{{node.onlineNum}}</span>
                <span class="badge" ng-hide="node.offlineNum==0&&node.onlineNum>0">{{node.offlineNum}}</span>
                <span class="fa fa-check" ng-if="node.id==groupActive.id"></span>      //icon
             </span>
          </treecontrol>
`
##  UI
![image](https://user-images.githubusercontent.com/24260845/38969463-0a1721b8-43c3-11e8-8b2a-60d844541603.png)
